api/src/main/java/org/openmrs/OrderSet.java

```diff
@@ -96,7 +98,25 @@ public void setOperator(Operator operator) {
 	public void setOrderSetMembers(List<OrderSetMember> orderSetMembers) {
 		this.orderSetMembers = orderSetMembers;
 	}
-	
+
+	/**
+	 * Gets the category
+	 *
+	 * @return the category
+	 */
+	public Concept getCategory () {
```

Comment:
Hi @dkayiwa after looking through the error logs, I noticed that the setter and getter in this class was referring to concept rather than the category field, I think I have changed this to the right code